Shared/Location/CLLocationManager+OneShotLocation.swift

```diff
@@ -28,13 +28,44 @@ enum OneShotError: Error, Equatable {
 }
 
 internal struct PotentialLocation: Comparable, CustomStringConvertible {
+    static var desiredAccuracy: CLLocationAccuracy { 100.0 }
+    static var invalidAccuracyThreshold: CLLocationAccuracy { 1500.0 }
+    static var desiredAge: TimeInterval { 30.0 }
+    static var invalidAgeThreshold: TimeInterval { 600.0 }
+
+    enum Quality {
+        case invalid
+        case meh
+        case perfect
+    }
+
     let location: CLLocation
+    let quality: Quality
 
-    static var desiredAccuracy: CLLocationAccuracy { 200.0 }
-    static var desiredAge: TimeInterval { 30.0 }
+    init(location: CLLocation) {
+        self.location = location
+
+        if location.coordinate.latitude == 0 || location.coordinate.longitude == 0 {
+            // iOS 13.5? seems to occasionally report 0 lat/long, so ignore these locations
+            Current.Log.error("Location \(location.coordinate) was super duper invalid")
```

Comment:
Do you want to log this event to Crashlytics so we can try to determine how widespread the issue is?

Reply:
yeah that's a good idea, I'll do that